src/components/CreateContact/CreateContact.js

```diff
@@ -0,0 +1,119 @@
+import React, { useState } from 'react'
+import styled from 'styled-components/macro'
+
+export default function CreateContact( onSubmit ) {
+
+    const [formData, setFormData] = useState('')
+    const [disabledButton, setDisabledButton] = useState(true)
+    return (
+        <div>
+            <FormStyled onSubmit={handleSubmit}>
+            <LabelWrapper>
+                <LabelStyled>
+                    Vorname
+                    <input 
+                    autoFocus
+                    name="firstName"
+                    value={formData.firstName}
+                    onChange={handleChange}
+                    placeholder="Max"
+                    />  
+                </LabelStyled>
+                <LabelStyled>
+                    Nachname
+                    <input 
+                    name="lastName"
+                    value={formData.lastName}
+                    onChange={handleChange}
+                    placeholder="Mustermann"
+                    />  
+                </LabelStyled>
+            </LabelWrapper>
+                <LabelStyled>
+                    Unternehmen
+                    <input 
+                    name="company"
+                    onChange={handleChange}
+                    placeholder="Universal GmbH"
+                    />  
+                </LabelStyled>
+                <LabelStyled>
+                    Telefon
+                    <input 
+                    name="phone"
+                    />  
+                </LabelStyled>
+                <LabelStyled>
+                    E-Mail
+                    <input 
+                    name="mail"
+                    />  
+                </LabelStyled>
```

Comment:
Have a look here: [mdn](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/input) you can pass some types for input element. type=email. it's better for accessibilty for blind people for example.

Reply:
I added the types to the inputs. Thanks!